Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -1757,6 +1840,152 @@ def send_mail_command():
     return sent_mail_to_entry('Email sent:', [result], emailto, emailfrom, cc, bcc, body, subject)
 
 
+def forwarding_address_add_command():
+    """
+    Creates a forwarding address.
+    """
+
+    args = demisto.args()
+    forwarding_email = args.get('forwarding_email', '')
+    user_id = args.get('user_id', '')
+    request_body = {'forwardingEmail': forwarding_email}
+    service = get_service(
+        'gmail',
+        'v1',
+        ['https://www.googleapis.com/auth/gmail.settings.sharing'],
+        delegated_user=user_id)
+    result = service.users().settings().forwardingAddresses().create(userId='me', body=request_body).execute()
+    readable_output = "Added forwarding address {0} for {1} with status {2}.".format(forwarding_email, user_id,
+                                                                                     result.get('verificationStatus',
```

Comment:
What could be the status fo result.get('verificationStatus')?

Reply:
Status retrieved from API response.